packages/core/package.json

```diff
@@ -3,9 +3,10 @@
   "version": "2.0.4",
   "description": "Priceline Design System",
   "main": "dist/index.cjs.js",
-  "module": "dist/index.esm.js",
+  "module": "dist/index.js",
+  "sideEffects": false,
   "scripts": {
-    "prepare": "rollup --config",
+    "prepare": "babel src/ --out-dir dist --ignore spec.js,story.js; rollup --config",
```

Comment:
Wonder why you prefer `;` over `&&`. Read a bit on topic in a somewhat [older stackexchange post](https://unix.stackexchange.com/questions/37069/what-is-the-difference-between-and-when-chaining-commands). Would `&&` work also? If for `;` what returned value from first `babel` would the second `rollup` command depend on?

Reply:
I actually don't think it really matters here. I'm open to either.